modules/hw-installing-amq-interconnect-messaging-bus.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * monitoring/using-rfhe.adoc
+
+:_content-type: PROCEDURE
+[id="hw-installing-amq-interconnect-messaging-bus_{context}"]
+= Installing the AMQ messaging bus
+
+To pass Redfish hardware event notifications between publisher and subscriber on a node, you must install and configure an AMQ messaging bus to run locally on the node. You do this by installing the AMQ Interconnect Operator for use in the cluster.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+* Install the AMQ Interconnect Operator to its own `amq-interconnect` namespace. See link:https://access.redhat.com/documentation/en-us/red_hat_amq/2021.q1/html/deploying_amq_interconnect_on_openshift/adding-operator-router-ocp[Installing the AMQ Interconnect Operator].
+
+.Verification
+
+. Verify that the AMQ Interconnect Operator is available and the required pods are running:
++
+[source,terminal]
+----
+$ oc get pods -n amq-interconnect
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                    READY   STATUS    RESTARTS   AGE
+amq-interconnect-645db76c76-k8ghs       1/1     Running   0          23h
+interconnect-operator-5cb5fc7cc-4v7qm   1/1     Running   0          23h
+----
+
+. Verify that the required `hw-event-proxy` hardware event producer pod is running in the `openshift-hw-events` namespace:
```

Comment:
With product name change, I think 

1. all instances of hw-events should be changed to baremetal-events. 
2. all instances of hw-event-proxy should be changed to baremetal-event-relay

@nishant-parekh  @jzding please ensure the contents in repo will be changed to align with product name.

Reply:
the changes for these are covered in this PR correct?
https://github.com/openshift/openshift-docs/pull/45898